static/app/views/eventsV2/table/quickContext.tsx

```diff
@@ -287,11 +303,47 @@ function ReleaseContext(props: ReleaseContextProps) {
   }
 
   return (
-    <Fragment>
+    <Wrapper data-test-id="quick-context-hover-body">
       {renderReleaseDetails()}
       {renderIssueCountAndAuthors()}
       {renderLastCommit()}
-    </Fragment>
+    </Wrapper>
+  );
+}
+
+function EventContext(props: BaseContextProps) {
+  let stackTrace: StacktraceType | null = null;
+  const {isLoading, isError, data} = useQuery<Event>(
+    [
+      `/organizations/${props.organization.slug}/events/${props.dataRow['project.name']}:${props.dataRow.id}/`,
+    ],
+    undefined,
+    {
+      staleTime: fiveMinutesInMs,
+      retry: false,
+    }
+  );
+
+  if (isLoading || isError) {
+    return <NoContext isLoading={isLoading} />;
```

Comment:
Nit: a bit confusing why the error and loading states are being combined here

Reply:
It's always one or the other so I made a small component [here](https://github.com/getsentry/sentry/blob/d78db4675fb00793a113c2111eeb7c84a874fd18/static/app/views/eventsV2/table/quickContext.tsx#L138) that has the ternary showing the loading/error message to avoid duplication across the different context components in this file.